### ISSUE TITLE
Can't autocomplete for files in subfolder

### ISSUE THREAD

**Issue by AllanLRH** [liamcain/AutoFileName#39]
Filename autocompletion is a neat feature, but is it possible to autocomplete filenames from a subfolder?
I'd like to use this plugin with LaTeX, where I have code like `\includegraphics[]{figs/someIllustration.pdf}`, and autocompleting figure name would be awesome.
Tried putting this in my settings file (the scope of the curly braces is `meta.group.braces.tex`):

``` json
"auto_complete_triggers":
    [
      {
         "characters": "figs/",
         "selector": "meta.group.braces.tex"
      },
      {
         "characters": "/",
         "selector": "meta.group.braces.tex"
      }
    ]
```

**Comment by liamcain** [comment]
Sorry I never responded to this. Did you ever get if figured out? AutoFileName autocompletes subfolders by default so there should be no need to add those triggers. Let me know if you're still having trouble.

**Comment by AllanLRH** [comment]
Nope. But I made a GIF showing the problem... since it should work, I want to be sure there's no miscommunication about the problem :)

![output_optimized](https://f.cloud.github.com/assets/2641327/1696907/655b05d8-5ecd-11e3-957a-355af5f371d2.gif)

**Comment by josh-azrael** [comment]
I have the same issue.

**Comment by czhang03** [comment]
latexing supports that, if you reenter the braket
